src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
I guess I didn't actually answer the question... what I put in issue 17001 is that this library should have a stock pool of pregenerated DSA keys that it uses for (random or not) importing for conformance testing.  *Generation* is blocked on macOS, not the algorithm itself (though Apple's version is stuck in pre-2009's SHA-1-only mode).